portfolio/src/main/java/com/google/sps/comment/repository/CommentRepositoryFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.comment.repository;
+
+import com.google.sps.comment.Comment;
+import com.google.sps.comment.repository.impl.DatastoreCommentRepository;
+import org.jetbrains.annotations.Nullable;
+
+public class CommentRepositoryFactory{
```

Comment:
java doc and spaces

https://google.github.io/styleguide/javaguide.html#s4.6-whitespace

Reply:
done